db-commons/src/main/scala/org/bitcoins/db/DbCommonsColumnMappers.scala

```diff
@@ -166,6 +184,102 @@ abstract class DbCommonsColumnMappers {
     MappedColumnType
       .base[TxoState, String](_.toString, TxoState.fromString(_).get)
   }
+
+  implicit val hdAccountMapper: BaseColumnType[HDAccount] = {
+    MappedColumnType.base[HDAccount, String](
+      _.toString,
+      BIP32Path.fromString(_).asInstanceOf[HDAccount])
+  }
+
+  implicit val dlcPubKeysMapper: BaseColumnType[DLCPublicKeys] = {
```

Comment:
I think [best practice](https://stackoverflow.com/questions/444251/how-to-store-a-list-in-a-db-column) is to not store this in a single column but rather either each pub key gets a column or it goes in its own table